planner/core/point_get_plan.go

```diff
@@ -294,7 +301,13 @@ func (p *BatchPointGetPlan) AccessObject() string {
 			buffer.WriteString(", index:" + p.IndexInfo.Name.O + "(")
 		}
 		for i, idxCol := range p.IndexInfo.Columns {
-			buffer.WriteString(idxCol.Name.O)
+			if tblCol := p.TblInfo.Columns[idxCol.Offset]; tblCol.Hidden {
+				exprStr := strings.Replace(tblCol.GeneratedExprString, "`", "", -1)
+				exprStr = strings.Replace(exprStr, " ", "", -1)
```

Comment:
Replace " " to "" is dangerous. What if concat("     ", "a")?

Reply:
![image](https://user-images.githubusercontent.com/30121694/96228990-e8e30500-0fc8-11eb-98b5-53d8239757a0.png)

Reply:
Maybe we only replace \` ?
The result like this `table:t, index:expr_idx_e(b + a)`

Reply:
![image](https://user-images.githubusercontent.com/30121694/96229819-e0d79500-0fc9-11eb-839a-cc441199c5c7.png)

Reply:
What if ('`' + ' ')?
@lzmhhh123 Any suggestion?

Reply:
@wjhuang2016 @lzmhhh123 I think we can't replace any symbols. Just show `column.GeneratedExprString` field.

Reply:
I agree with you.

Reply:
a new commit to reverts commit 96f45bca. PTAL